### PR TITLE
add const and comp

### PR DESCRIPTION
**Pull request by newreq7**

### CODE REVIEW

File: src/components/cleanButton.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const CleanButton = () => {
```
Comment (lina-startuporaft) [inline src/components/cleanButton.jsx]: here you should pass "props"
```suggestion
export const CleanButton = ({cleanFields}) => {
```

**Review by lina-startuporaft — CHANGES_REQUESTED** [review]

File: src/components/cleanButton.jsx (hunk 1 above)
Comment (lina-startuporaft) [reply src/components/cleanButton.jsx:1]: Далее тебе нужно соответственно передать эту функцию в onclick кнопки ниже

File: src/components/cleanButton.jsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export const CleanButton = () => {
+export const CleanButton = (cleanFields) => {
```
Comment (lina-startuporaft) [inline src/components/cleanButton.jsx]: not sure. it works? I think you should use {}.

**Review by lina-startuporaft — CHANGES_REQUESTED** [review]

File: src/components/cleanButton.jsx (hunk 2 above)
Comment (lina-startuporaft) [reply src/components/cleanButton.jsx:1]: if you don't use {}, you can write (props), then props.cleanFields

**Review by lina-startuporaft — CHANGES_REQUESTED** [review]

File: src/components/cleanButton.jsx (hunk 2 above)
Comment (newreq7) [reply src/components/cleanButton.jsx]: I fixed it. Now it works
Thnx!

Comment (lina-startuporaft) [reply src/components/cleanButton.jsx]: I don't see a new commit...

**Review by lina-startuporaft — APPROVED** [review]

**Pull request merged**